File: src/main/java/YourListsPage.java
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;
import org.openqa.selenium.support.PageFactory;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class YourListsPage extends BasePage {

    private WebDriverWait wait;
    private WebDriver driver;
    @FindBy(xpath = "//*[text()='Megunhatatlan']")
    private WebElement megunhatatlan;
    @FindBy(xpath = "//*[text()='Inglourious Basterds']")
    private WebElement checkMovie;

    public YourListsPage() {
        this.driver = getDriver();
        this.wait = getWait();
        PageFactory.initElements(driver, this);
    }

    public void clickOnMegunhatatlan(){
        wait.until(ExpectedConditions.elementToBeClickable(megunhatatlan));
        megunhatatlan.click();
    }

    public boolean checkMovie() {
        wait.until(ExpectedConditions.elementToBeClickable(checkMovie));
        return checkMovie.isDisplayed();
    }
}

File: src/main/java/your_lists_page.py
from selenium.webdriver.common.by import By

from base_page import BasePage


class YourListsPage(BasePage):

    megunhatatlan = (By.XPATH, "//*[text()='Megunhatatlan']")
    check_movie = (By.XPATH, "//*[text()='Inglourious Basterds']")

    def __init__(self, driver):
        super().__init__(driver)

    def click_on_megunhatatlan(self):
        self.driver.find_element(*self.megunhatatlan).click()
